labgrid-web-client/src/app/place/place.component.css

```diff
@@ -12,3 +12,18 @@
     margin-right: 10px;
     margin-bottom: 10px;
 }
+
+.place-table {
+    width: 800px;
+    margin-top: 16px;
+    column-width: 400px !important;
+}
+
+.place-table > td{
```

Comment:
This rule has no effect because it affects td element one level below place-table. But we always have a tr element between. To make it work you should remove the ">" symbol.